packages/components/src/Layout/SkipLinks/SkipLinks.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
+
+import Icon from '../../Icon';
+import I18N_DOMAIN_COMPONENTS from '../../constants';
+import getDefaultT from '../../translate';
+
+import theme from './SkipLinks.scss';
+
+function SkipTo({ href, label }) {
+	return (
+		<a href={href}>
+			<span className={theme.icon}>
+				<Icon transform="rotate-270" name="talend-arrow-left" />
+			</span>
+			<span className={theme.text}>{label}</span>
+		</a>
+	);
+}
+SkipTo.propTypes = {
+	href: PropTypes.string.isRequired,
+	label: PropTypes.string.isRequired,
+};
+
+function SkipLinks({ mainId, navigationId, t }) {
```

Comment:
The idea is not to display every part of the page. To do that you have SR, with the right attributes (role search/search-input).

Reply:
It will be nice to access nav, main content **and** search. 
But TDP is the only one with a search bar, and I will not use this component for now. 
Let's see that in the second run.